### PR TITLE
Pin go-msgpack to a version compatible with Consul

### PR DESCRIPTION
**Pull request by kisunji**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -34,7 +34,7 @@ require (
 	github.com/google/uuid v1.3.0 // indirect
 	github.com/hashicorp/errwrap v1.0.0 // indirect
 	github.com/hashicorp/go-cleanhttp v0.5.1 // indirect
-	github.com/hashicorp/go-msgpack v1.1.5 // indirect
+	github.com/hashicorp/go-msgpack v0.5.5 // indirect; DO NOT UPDATE
```
[the remark this answers is not in the document]
Comment (kisunji) [reply go.mod:37]: No more need for this comment; v1.1.5 has been retracted

**Comment by dgregoire** [comment]
Closing this.

**Review by lmorton-tran — APPROVED** [review]
Thanks! Please wait for approval from @dgregoire first.

**Comment by kisunji** [comment]
Just closing out of precation; @dgregoire is looking into removing the dependency since `v0.5.5` is not compatible with scada brokers

**Comment by dgregoire** [comment]
> Just closing out of precaution; @dgregoire is looking into removing the dependency since `v0.5.5` is not compatible with scada brokers

False negative test, the problem is elsewhere. Interop looks ok.

**Pull request closed without merging**